local/src/core/http.cc

```diff
@@ -911,21 +911,18 @@ Session::write(TextView view)
   return zret;
 }
 swoc::Errata
-Session::send_proxy_header(swoc::IPEndpoint const *real_target, ProxyProtocolVersion pp_version)
+Session::send_proxy_msg(ProxyProtocolMsg const &pp_msg)
 {
   swoc::Errata errata;
-  // get source/local endpoint
-  struct sockaddr addr;
-  socklen_t len = sizeof(addr);
-  getsockname(_fd, &addr, &len);
-  swoc::IPEndpoint source_endpoint{&addr};
-  // construct the PROXY header
-  ProxyProtocolUtil ppUtil(source_endpoint, *real_target, pp_version);
   swoc::LocalBufferWriter<MAX_PP_HDR_SIZE> w;
-  auto pp_serialize_errata = ppUtil.serialize(w);
+  auto pp_serialize_errata = pp_msg.serialize(w);
   errata.note(std::move(pp_serialize_errata));
   // send the data
-  errata.note(S_DIAG, "Sending PROXY header from {} to {}", source_endpoint, *real_target);
+  errata.note(
+      S_DIAG,
+      "Sending PROXY header:\nsource address: {}\ndestination address: {}",
```

Comment:
Line breaks can make parsing more difficult. Let's remove the `\n` from this.

Reply:
Removed the `\n`